com.github.torikulhabib.niki.yml

```diff
@@ -0,0 +1,59 @@
+app-id: com.github.torikulhabib.niki
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: com.github.torikulhabib.niki
+finish-args:
+  - '--share=network'
+  - '--talk-name=org.freedesktop.thumbnails.Thumbnailer1'
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=pulseaudio'
+  - '--device=all'
+  - '--filesystem=xdg-videos'
+cleanup:
+  - '*.a'
+  - '*.la'
+  - '/include'
+modules:
+  - name: libgssdp-1.2-0
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/GNOME/gssdp.git
+        tag: gssdp-1.2.3
+  - name: libgupnp-1.2-0
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/GNOME/gupnp.git
+        tag: gupnp-1.2.4
+  - name: libgupnp-av-1.0-2
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/GNOME/gupnp-av.git
+        tag: gupnp-av-0.12.11
+  - name: libcanberra-0.30
```

Comment:
this is in the shared-modules https://github.com/flathub/shared-modules/ you should pull it in from there please.